frontend/src/models/associations/library.ts

```diff
@@ -19,13 +19,13 @@ export interface Loan {
 export interface Loanable {
   id: number;
   userLoan: null | (Omit<Loan, "loanable"> & { loanable: number });
-  numberOfPending: number;
+  numberOfPendingLoans: number;
   name: string;
   description?: string;
   image?: string;
   comment?: string;
   library: Library;
-  status: "AVAILABLE" | "BORROWED" | "REQUESTED";
+  status: "AVAILABLE" | "BORROWED";
```

Comment:
Won't `REQUESTED` be able with the admin UI ?

Reply:
See `AssociationLibraryManagement`

Reply:
Maybe one confusing thing here should be explained:
- The data, whether in the backend or the frontend side, only has the `AVAILABLE` and `BORROWED` status (you can see this in `library.ts` and in `models/library.py`).
- However, the _filter_ does not use the same convention (which is confusing I admit): you have three values, `AVAILABLE`, `BORROWED` and `REQUESTED`, to help separating the “available but requested” and “available but not requested” states, useful for the admins.

Reply:
Write a comment then please

Reply:
Well it's explained in the filters… and in the UI when necessary

Reply:
But this could be better I'll change it